docs/docsite/rst/network/user_guide/platform_index.rst

```diff
@@ -82,5 +82,7 @@ Settings by Platform
 | OS that supports  | ``<network-os>``        |             | ✓       |         | ✓        |
 | Netconf*          |                         |             |         |         |          |
 +-------------------+-------------------------+-------------+---------+---------+----------+
+| Pluribus Netvisor | ``netvisor``            | ✓           |         |         |          |
++-------------------+-------------------------+-------------+---------+---------+----------+
 
```

Comment:
It's not obvious, but the rest of this table is in alphabetical order, except the last "OS that supports Netconf*" row. Please move the Pluribus row up so that it is alphabetical with the other platforms.